### PR TITLE
[cache] Don't prepopulate if masquerade cache file is empty

### PR DESCRIPTION
**Pull request by soltzen**
Closes https://github.com/getlantern/lantern-internal/issues/5721

### CODE REVIEW

File: cache.go [hunk 1]
```diff
@@ -22,36 +22,45 @@ func (d *direct) initCaching(cacheFile string) int {
...
-				select {
-				case d.cached <- m:
-					// submitted
-					cache = append(cache, m)
-				default:
-					// channel full, that's okay
-				}
+	if len(bytes) == 0 {
+		// This can happen if the file is empty or just not there
+		log.Debugf("ignorable error: Cache file is empty")
```
Comment (oxtoacart) [inline cache.go]: Can just use `log.Debug`

**Review by oxtoacart — APPROVED** [review]

**Comment by oxtoacart** [comment]
@soltzen I went ahead and pushed a fix for the unit test failure.

File: cache.go (hunk 1 above)
Comment (soltzen) [reply cache.go]: Fixed

**Comment by soltzen** [comment]
Thanks for the quick review, @oxtoacart  🚀

**Pull request merged**